content/20.results.md

```diff
@@ -5,54 +5,49 @@ The ISCB Fellows set contained the complete set of Fellows named (2009--2019).
-
-In version 3, instead of weighting all articles equally as in the earlier versions, we used citation count to weight articles to control for the differential impact of research contributions.
-Using citation counts has key limitations: female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
-Furthermore, the act of being honored, particularly with a keynote at an international meeting, could lead to work being more recognized and cited, which would reverse the arrow of causality.
-In [version 4](https://greenelab.github.io/iscb-diversity-manuscript/), we returned to the equal weight for all articles.
-Parallel analyses for the other versions are available in supplementary figure S1.
-
-Finally, in all versions of the analysis, rather than applying a hard assignment for each prediction, we analyzed the raw prediction probability values to capture the uncertainty of the prediction model.
-Although we expect our estimates of disparities for citation-weighted analyses to be conservative, through each analysis, the overall findings remained consistent.
-Examining the literature with and without citation weighting, we learned that disparities exist and these disparities are large enough to overcome existing disparities in citation patterns.
+Study framework. We extracted full names and affiliations of the last authors of 176,110 computational biology PubMed articles and those of 412 honorees. We estimated the gender, name-origin group and country of affiliation of each scientist and compared the probability values between these two groups (see Methods).
+](https://raw.githubusercontent.com/greenelab/iscb-diversity/d84aa95ea02fff615c89668a20a957298df855ed/figs/workflow.png){#fig:workflow width="90%"}
 
 ### Similar gender proportion between ISCB's honorees and the field
 
 We observed a gradual increase of the proportion of predicted female authorships, arriving at an average of approximately 28% in 2017-2019 (Fig. {@fig:gender_breakdown}, left).
 In recent years, ISCB Fellows and keynote speakers appear to have similar gender proportions compared to the population of authors published in computational biology and bioinformatics journals (averaged around 30% in the last three years, Fig. {@fig:gender_breakdown}, right).
-Examining each honor category, we observed an increasing trend of honorees who were women, especially in the group of ISCB Fellows (see [notebook](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1)), which markedly increased after 2015.
+Examining each honor category, we [observed](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1) an increasing trend of honorees who were women, especially in the group of ISCB Fellows, which markedly increased after 2015.
```

Comment:
Does this need to refer to the notebook number like the others do?

Reply:
I don't think so. This is also very minor. My goal was for it to still make sense even when the hyperlink disappears.

Reply:
What about: "... we observed in [10.visualize-gender] an increasing..."? It feels like it should refer somewhere in the sentence if the hyperlink goes away.

Reply:
Sorry, I just now see this. OK I'll add in a separate commit.

Reply:
dde6d54a542ee8b05e58ca9cfc82e7340da98ae9